sass/_contact-form.scss

```diff
@@ -0,0 +1,42 @@
+.contentTitle {
+	font: $fancyFont;
+	text-align: center;
+}
+.contactIntro {
+	font: $fancyFont;
+	text-align: center;
+	margin: 0px 300px 0px 300px;
+}
+.googleButton {
+	background-color: $fccblue;
+	border-radius: 18px;
+	display: inline-block;
+	cursor: pointer;
+	color: #FFFFFF;
+	font: $fancyFont;
+	font-size: 20px;
+	margin: 30px 0px 0px 600px;
+	padding: 30px 50px;
+	text-decoration: none;
+	position: relative;
+}
+.googleButton:hover {
+	background-color: $fccyellow;
+	color: $fccGreen;
+}
+.googleButton:after {
+	content: '';
+	width: 0;
+	height: 0;
+	border: 30px solid;
+	text-align: center;
+	position: absolute;
+	border-color: $fccblue transparent transparent $fccblue;
+}
+.googleButton:hover::after {
```

Comment:
Though double colon is valid, it's inconsistent with the codebase, keep it to single colon.